web/src/components/molecules/Schema/ModelCreationModal/index.tsx

```diff
@@ -71,6 +71,9 @@ const ModelCreationModal: React.FC<Props> = ({
         <Form.Item
           name="key"
           label={t("Model key")}
+          extra={t(
+            "Model key must be unique and at least 5 characters, and can only contain letters, numbers, underscores and dahses.",
```

Comment:
Same as field key, so please update here with the field key one (replacing field key with model key of course)

Reply:
@KaWaite , changed.